### PR TITLE
Gh#2 7

### PR DESCRIPTION
**Pull request by maximorlenco**
Plz check

### CODE REVIEW

File: src/Main.java [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+        System.out.println(getAdresses(persons));
+
+    }
+
+    public static List<Address> getAdresses(List<Person>person){
+        List<Address> adres=new ArrayList<>();
+        for(Person s:person){
+            adres.add(s.getAdress());
+        }
+        return adres;
```
Comment (sergeylukichev) [inline src/Main.java]: lets make it with stream api and without loops

File: src/nameInRange4.java [hunk 2]
```diff
@@ -0,0 +1,25 @@
+import java.util.ArrayList;
+import java.util.List;
+//Задача №2
+public class nameInRange4 {
+    public static void main(String[] args) {
+        List<String> name = new ArrayList<>();
+        name.add("Ivan");
```
Comment (sergeylukichev) [inline src/nameInRange4.java]: Since this is a list with many names, the correct name for it would be "names".

File: src/nameInRange4.java [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+        name.add("Maria");
+        name.add("Stephan");
+        name.add("John");
+        name.add("Amalia");
+        System.out.println(checkLenght(name));
+    }
+    public static List<String> checkLenght(List<String> befor ){
+        List<String> nameLenghtFinal = new ArrayList<>();
```
Comment (sergeylukichev) [inline src/nameInRange4.java]: "Length", not "Leght".

File: src/nameInRange4.java [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+        for(int i = 0;i<befor.size();i++){
+            if(befor.get(i).length()>4||befor.get(i).length()<4){
+                nameLenghtFinal.add(befor.get(i));
+            }
+        }
```
Comment (sergeylukichev) [inline src/nameInRange4.java]: A bit overcomplicated, with streams would be easier. 

Even with the loop the check for `befor.get(i).length() != 4` would be simpler.

Comment (maximorlenco) [reply src/nameInRange4.java]: public static List<String> checkLength2(List<String> befor ){
       List<String> after = befor.stream().filter(a ->a.length()!=4).collect(Collectors.toList());
        return after;
    }

File: src/Main.java (hunk 1 above)
Comment (maximorlenco) [reply src/Main.java]: #6  #5

**Comment by maximorlenco** [comment]
https://github.com/maximorlenco/Homework/pull/4/commits/b4195d3116e8623681b75c6235124157bbd5505e

ср, 3 окт. 2018 г. в 17:31, Sergey Lukichev <notifications@github.com>:

**Pull request merged**